application/src/main/resources/db/V5__Add_Warn_System.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE warn_system
+(
+    /**
+     * This was taken from https://github.com/YusufsDiscordbot/Yusuf-s-Moderation-Bot and changed.
+     */
+    userid         BIGINT NOT NULL UNIQUE PRIMARY KEY,
+    guild_id       BIGINT NOT NULL,
+    warn_reason    TEXT DEFAULT 'This user has been warned for breaking the rules.',
+    is_warned      BIT DEFAULT FALSE,
+    warning_amount INTEGER DEFAULT 0
+)
```

Comment:
The way you realized those tables doesnt look very useful to me, honestly. It is also overfly fragile, instable and has a lot of repetition.

Instead, you should save single records for each event and not update them on the fly. Also, you can have a single database table for all moderation actions and then use the `ModerationUtils.Action` enum type to differentiate.

As in, you should rather have sth like:
* case_id (primary key, autoincrement)
* user_id (not unique)
* guild_id
* reason
* action_type
* timestamp

Reply:
I don't know how to have one single database table